tests/driver_bme680/tests/01-run.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python3
```

Comment:
It seems to be the test file from `tests/pkg_u8g2`.

Reply:
Indeed, removed from now.